docs/usage/key-concepts/dashboard.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
Do we want images/screenshots in this document? See the bottom post in the PR for my comments/thoughts.

Resolve this PR comment once we have reached a decision.

Reply:
Let's merge without them, but definitely it would be better to have them added